internal/nix/nix.go

```diff
@@ -46,7 +46,7 @@ func getExpectedMachineId(path, hostname string) (machineId string, err error) {
 }
 
 func runNixCommand(args []string, stdout, stderr io.Writer) (err error) {
-	commonArgs := []string{"--extra-experimental-features", "nix-command", "--extra-experimental-features", "flakes"}
+	commonArgs := []string{"--extra-experimental-features", "nix-command", "--extra-experimental-features", "flakes", "--accept-flake-config"}
```

Comment:
I'm wondering if comin should enable this flag by default becuse of security issues.
However, if an attacker is able to introduce `extra-substituters`, this attacker could also first change the global nix config...